src/m365/planner/commands/bucket/bucket-get.ts

```diff
@@ -0,0 +1,215 @@
+import { PlannerBucket } from '@microsoft/microsoft-graph-types';
+import { Logger } from '../../../../cli';
+import { AxiosRequestConfig } from 'axios';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { validation } from '../../../../utils';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id?: string;
+  title?: string;
+  planId?: string;
+  planName?: string;
+  ownerGroupId?: string;
+  ownerGroupName?: string;
+}
+
+class PlannerBucketGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.BUCKET_GET;
+  }
+
+  public get description(): string {
+    return 'Gets the Microsoft Planner bucket in a plan';
```

Comment:
We should ensure that we are using the same description in the documentation and in the command